docs/src/pages/components/scrollview/demo.tsx

```diff
@@ -1,41 +1,28 @@
-import * as React from 'react';
-import { Image, ScrollView, ScrollViewProps } from '@aws-amplify/ui-react';
+import { Image, ScrollView, useTheme } from '@aws-amplify/ui-react';
+
 import { Demo } from '@/components/Demo';
-import { useScrollViewProps } from './useScrollViewProps';
-import { ScrollViewPropControls } from './ScrollViewPropControls';
-import { demoState } from '@/utils/demoState';
 
-const propsToCode = (props: ScrollViewProps) => {
-  return `<ScrollView
-  orientation="${props.orientation}"
-  height="300px"
-  width="400px"
->
+export const ScrollViewDemo = () => {
+  const {
+    tokens: { colors },
+  } = useTheme();
+  return (
+    <Demo
+      code={`
```

Comment:
Can we move the code outside and use a variable to store it?

Reply:
fixed